services/apps/premium/organizations_enrichment_worker/src/repos/tenant.repo.ts

```diff
@@ -0,0 +1,47 @@
+import { TenantPlans } from '@crowd/types'
+import { Logger } from '@crowd/logging'
+import { DbStore, RepositoryBase } from '@crowd/database'
+
+export interface IPremiumTenantInfo {
+  id: string
+  plan: TenantPlans
+}
+
+export class TenantRepository extends RepositoryBase<TenantRepository> {
+  constructor(dbStore: DbStore, parentLog: Logger) {
+    super(dbStore, parentLog)
+  }
+
+  async getPremiumTenants(perPage: number, lastId?: string): Promise<IPremiumTenantInfo[]> {
+    if (lastId) {
```

Comment:
_Very much nitpicking:_ 
We can simplify this function by adding a lastIdFilter with default empty string, and when lastId exists the last filter part of the query. Something like 
`const lastIdFilter = lastId ? ' and id > $(lastId) ' : ''`
since lastId will be undefined if wasn't sent, it'll also not break the replacements